openedx/core/lib/request_utils.py

```diff
@@ -163,42 +161,42 @@ def process_exception(self, request, exception):
 # .. toggle_use_cases: opt_out
 # .. toggle_creation_date: 2021-03-11
 
-# .. toggle_name: EXPECTED_ERRORS[N]['LOG_ERROR']
+# .. toggle_name: IGNORED_ERRORS[N]['LOG_ERROR']
 # .. toggle_implementation: DjangoSetting
 # .. toggle_default: False
 # .. toggle_description: If True, the error will be logged with a message like: "Expected error ...".
 # .. toggle_use_cases: opt_in
 # .. toggle_creation_date: 2021-03-11
 
-# .. toggle_name: EXPECTED_ERRORS[N]['LOG_STACK_TRACE']
+# .. toggle_name: IGNORED_ERRORS[N]['LOG_STACK_TRACE']
 # .. toggle_implementation: DjangoSetting
 # .. toggle_default: False
 # .. toggle_description: If True, the stacktrace will be included with the logging message.
 # .. toggle_warning: Requires ``LOG_ERROR`` to be set to True, otherwise this value will be ignored.
 # .. toggle_use_cases: opt_in
 # .. toggle_creation_date: 2021-03-11
 
-# .. setting_name: EXPECTED_ERRORS[N]['REASON_EXPECTED']
+# .. setting_name: IGNORED_ERRORS[N]['REASON_IGNORED']
 # .. setting_default: None
 # .. setting_description: Required string explaining why the error is expected and/or ignored for documentation
 #     purposes.
 
 
-# Warning: do not access this directly, but instead use _get_expected_error_settings_dict.
+# Warning: do not access this directly, but instead use _get_ignored_error_settings_dict.
 # EXPECTED ERRORS Django setting is processed and stored as a dict keyed by ERROR_MODULE_AND_CLASS.
```

Comment:
Please search file for “expected” to see if there are any missed references like this one.

Reply:
@robrap I think I already did it not missed.[u can check here](https://github.com/openedx/edx-platform/pull/33184/commits/6357bb0a1617aa34a640797bd7f646cc8b353c36)

Reply:
1. The line I have commented here begins with `# EXPECTED ERRORS Django...`, so you missed some cases.
2. If you look at [the full request_utils.py file with your changes](https://github.com/openedx/edx-platform/blob/e8cddb2f473ca95e3b74db211470168e3ba407c3/openedx/core/lib/request_utils.py), and search for "expected", you'll see other cases that were missed.
3. For each file you've updated in this PR, use "View File" to see the whole file, or search locally to see what else you may have missed.
Thank you.

Reply:
This line still begins with `# EXPECTED ERRORS Django ...`, and should be `# IGNORED_ERRORS Django ...`. 

I wrote this comment earlier and I asked you to search, and this still has not been corrected. This makes reviewing a frustrating experience.

Please do not ask me to review until all comments have been resolved. Thank you.